<launch>
  <!-- view generator -->
  <include file="$(find scitos_moveit_config)/launch/move_group.launch"/>
  <node name="ptu_fk" pkg="viper" type="ptu_fk"/>

  <rosparam param="roi">[[2.0,-8.0],[5.0,-8.0],[5.0,2.0],[2.0,2.0]]</rosparam>   
  <param name="inflation_radius" type="double" value="0.5" />  
  <param name="views_at_pose" type="int" value="8" />  

  <node name="octomap" pkg="octomap_server" type="octomap_server_node" args="$(find strands_morse)/bham/maps/cs_lg_obj_search_octomap.bt"/>

  <arg name="map" default="/projected_map"/>  
  <include file="$(find nav_goals_generator)/launch/nav_goals_generator.launch">
    <arg name="map" value="$(arg map)"/>
  </include>
</launch>